modules/ROOT/pages/access-developer-portal-task.adoc

```diff
@@ -1,27 +1,34 @@
-= Accessing the Developer Portal
+= Access the Developer Portal
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
```

Comment:
Scanning over this topic, I already see some organizational issues:

- This topic is called Access the Developer Portal, but it has subtopics Access (the) API Instance Portal and Access the Public Portal. Do these two portals comprise the Dev. portal? Clarify.

- That said, I think this topic can be a lot shorter and to the point:

1. To access a private portal, you need Portals Viewer access.
2. You can access any public portal.
3. Here is how you access a private portal.
4. Here is how you access a public portal.